downstream/modules/hub/con-enable-view-only.adoc

```diff
@@ -1,9 +1,10 @@
 // Module included in the following assemblies:
 // assembly-view-only-access.adoc
 
-
 [id="con-enable-view-only"]
 
+= Enabling view-only access for your {PrivateHubNameStart}
```

Comment:
Just curious... should {PrivateHubName} here instead of the one with "start" included (since it doesn't come at the beginning of the title)

Reply:
You are correct. Fixed 👍🏾